src/main/treeDataProviders/dependenciesTree/dependenciesTreeNode.ts

```diff
@@ -96,6 +96,7 @@ export class DependenciesTreeNode extends vscode.TreeItem {
 
     public shallowClone(): DependenciesTreeNode {
         let clone: DependenciesTreeNode = new DependenciesTreeNode(this.generalInfo, this.collapsibleState);
+        clone.label = this.label;
```

Comment:
Only this line needed. Please revert all other changes.

Reply:
Did you check it?

Reply:
yes